src/env.d.ts

```diff
@@ -21,6 +21,12 @@ declare global {
 		},
 	}
 
+	const OC : {
+		MimeType: {
+			getIconUrl: (mimetype: string) => string,
```

Comment:
According to server code:
```suggestion
			getIconUrl: (mimetype?: string) => string | undefined,
```
Should be fine, since you're providing a file as a fallback ayway